docs/nodes/providers/2_setting_up.md

```diff
@@ -42,10 +31,10 @@ git pull
 
 git checkout {version}
 
-make install
-
 source ~/.profile
 
+make install
+
```

Comment:
I had to re-source the shell during my installation after running 'make install' to install jprovd--so we shouldn't change this.

It's because the binary just got installed in the user's PATH so we re-source after running 'make install' to make the shell recognise that the PATH got updated? Am I making sense? haha

Reply:
I've had to run it before doing make install - maybe this is why you got the "install make" thing?

Reply:
It's mentioned to run it at the very end of this section

https://docs.jackaldao.com/docs/nodes/install

Which should take care of sourcing everything you need to build the daemon.

I run it again for the reason described above.

Reply:
clicking review button :)